### DISCUSSION TITLE
GPT4

### DISCUSSION THREAD

**Discussion by nomadphase** [nhaouari/obsidian-textgenerator-plugin#129] [General]
Hi Nizar,

Love your work on this project. Do you have an application in for GPT4 to be included within the plugin?
Is there anything that can be done to support that type of application?

Cheers,
Peter

**Comment by nhaouari** [comment]
if you access to gpt-4, it is available in v0.2.21

**Reply by circuitsnacks** [reply]
it seems since the update 20 hours ago there may be a bug?  I'm not having issues in vaults without the update. But with the update, it's broken for me.  Any suggestions?  

I was using 3.5turbo fine before the update that now includes gpt4.  Even if I keep the settings to 3.5turbo with the new update installed, I get no output.

**Comment by nomadphase** [comment]
Thank you, amazing work

I received this error message

> [!failure]- Failure 
>   Error: Request failed, status 404
>   
>   - app.js:1 new t
>     app://obsidian.md/app.js:1:717762
>   
>   - app.js:1 Jw
>     app://obsidian.md/app.js:1:717954
>   
>   - app.js:1 
>     app://obsidian.md/app.js:1:718631
>   
>   - app.js:1 
>     app://obsidian.md/app.js:1:235836
>   
>   - app.js:1 Object.next
>     app://obsidian.md/app.js:1:235941
>   
>   - app.js:1 a
>     app://obsidian.md/app.js:1:234680

**Reply by nhaouari** [reply]
Verify your eligibility to access the GPT-4 on the OpenAI Playground, as it is currently restricted to waitlisted users.

**Reply by davidcarnahan** [reply]
This is very interesting. I got the same error above so I changed it to GPT3.5 Turbo. I have GPT4 in their UI, but not in playground.
